build.cake

```diff
@@ -250,9 +250,9 @@ Task("TestCore")
 
     foreach (var testProject in testProjects)
     {
-        var logFile = System.IO.Path.Combine(logFolder, $"{testProject}-core-result.xml");
+        var logFile = System.IO.Path.Combine(logFolder, $"{testProject}-core-result.trx");
```

Comment:
dotnet no longer supports xml output, so we need to update CI builds to read TFX files